lib/internal/test_runner/test.js

```diff
@@ -579,8 +579,8 @@ class Test extends AsyncResource {
       await after();
       this.pass();
     } catch (err) {
-      try { await after(); } catch { /* Ignore error. */ }
       try { await afterEach(); } catch { /* test is already failing, let's ignore the error */ }
+      try { await after(); } catch { /* Ignore error. */ }
```

Comment:
@MoLow doesn't this mean we might call this twice and run them more than once in case calling `after` threw?